src/test/csi-neonsan/templates/controller-deploy.yaml

```diff
@@ -0,0 +1,150 @@
+kind: Deployment
+apiVersion: apps/v1
+metadata:
+  name: csi-neonsan-controller
+  namespace: {{ .Release.Namespace }}
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      role: "controller"
```

Comment:
The label is too simple and maybe conflict with other workloads.
You should add more labels, such as https://github.com/yunify/qingcloud-csi/blob/8366ba67b99799d725964775e5a4fdf244eeca78/deploy/disk/kubernetes/releases/qingcloud-csi-disk-v1.17.yaml#L5

Reply:
i add lables to avoid confilct